frontend/src/__mocks__/mockNimResource.ts

```diff
@@ -45,11 +45,20 @@ export const mockNimImages = (): ConfigMapKind =>
     },
   });
 
-export const mockNimInferenceService = (): InferenceServiceKind => {
+type NimInferenceService = {
+  namespace?: string;
+  displayName?: string;
+};
+
+export const mockNimInferenceService = ({
+  displayName = 'Test Name',
+  namespace = 'test-project',
+}: NimInferenceService): InferenceServiceKind => {
   const inferenceService = mockInferenceServiceK8sResource({
     name: 'test-name',
     modelName: 'test-name',
-    displayName: 'Test Name',
+    displayName: displayName || 'Test Name',
```

Comment:
You double defaulted? Why? Once in the signature of the function & one more here...